src/test/java/spoon/test/textBlocks/TextBlockTest.java

```diff
@@ -0,0 +1,49 @@
+package spoon.test.textBlocks;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import spoon.Launcher;
+import spoon.reflect.code.CtInvocation;
+import spoon.reflect.code.CtLiteral;
+import spoon.reflect.code.CtStatement;
+import spoon.reflect.code.CtTry;
+import spoon.reflect.declaration.CtClass;
+import spoon.reflect.declaration.CtMethod;
+
+public class TextBlockTest{
+	private Launcher setUpTest() {
+		Launcher launcher = new Launcher();
+		launcher.addInputResource("./src/test/resources/textBlock/TextBlockTestClass.java");
+		launcher.run();
+		return launcher;
+	}
+	
+	@Test
+	public void testTextBlock(){
```

Comment:
In Spoon every testcase has a comment `//contract: ...` describing the testcase in natural language